ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -37,6 +38,8 @@ Item {
 
             let formattedMessage = Utils.linkifyAndXSS(root.messageDetails.messageText);
 
+            isQuote = (formattedMessage.indexOf("blockquote") > -1);
```

Comment:
What do you think of adding a more restrictive check here? I'm thinking to check if the html table with class blockquote exists, not just "blockquote".

Checking only for "blockquote" means the "blockquote" becomes a formatter by itself.
Ex: Hello blockquote -> will become a quoted text

Reply:
I agree but also think the usage of table/tr/td is overkill here

Reply:
@alexjba the html table is removed (as per Lukas comment), now we only get the plain text starting with `blockquote:` so that we can identify it in QML, which is then removed - if you have  any better idea, please let know!

Reply:
I'm glad it's removed.
Imo any additional restriction in indexOf is a good thing. I like the idea Lukas had to use the <blockquote> html tag. This allows us to add a specific class to it (something like <blockquote class="StatusBlockQuote">). We can then filter for "<blockquote class="StatusBlockQuote">".

Reply:
Yup, that's a good idea imo